decidim-comments/app/frontend/support/schema.ts

```diff
@@ -59,11 +59,11 @@ export interface GetCommentsQuery {
     } | null,
   } | null;
   commentable: AddCommentFormCommentableFragment & {
-    // Whether the object can have new comments or not
+    // Wether the object can have new comments or not
```

Comment:
Mmmmm. Apparently a "wether" is a "castrated ram (a male sheep) or a castrated billy (a male goat)"... http://www.grammar-monster.com/easily_confused/weather_whether_wether.htm :laughing: